README.md

```diff
@@ -99,6 +99,7 @@ Run `make help` to see all the possible targets and variables:
     PSM                Page segmentation mode. Default: 6
     RANDOM_SEED        Random seed for shuffling of the training data. Default: 0
     RATIO_TRAIN        Ratio of train / eval training data. Default: 0.90
+    MIN_GT_FILES       Minimum number of GT files. Default: 10
 ```
```

Comment:
This looks like a relict from the original pull request.

Reply:
@kba, this commit is good but conflicts with my change which only fixed `TESSERACT_VERSION`.